### DISCUSSION TITLE
Keyball61 特定の行のキーが反応しない

### DISCUSSION THREAD

**Discussion by OQFFF** [Yowkees/keyball#265] [Q&A] [answered]
Keyball61 トラックボール側です。
表題の通り、上から4行目のキー全てが反応しません。ProMicroが原因だと思い、左右取り替えて試しましたが、何も変化はありませんでした。
ダイオードやキーソケットのはんだ状況が原因でしょうか?
![30_20230629221524](https://github.com/Yowkees/keyball/assets/104715416/6a7e756b-40d6-4ace-b053-ec6f5a2ec49a)

**Comment by OQFFF** [comment, marked as answer]
自己解決しました
基板側ROW3のはんだ不良が原因でした。
お騒がせしました。

**Comment by Yowkees** [comment]
ご連絡が遅くなり、大変申し訳ありませんでした。
ご自身で見直していただき、誠にありがとうございました。
